src/FilterBundle/Resources/config/services.yml

```diff
@@ -81,4 +81,14 @@ services:
 
   best_it_commercetools_filter.helper.enum_attribute_helper:
       class: BestIt\Commercetools\FilterBundle\Helper\EnumAttributeHelper
-      arguments: ['@best_it_commercetools_filter.request.client', '@cache.app', '%best_it_commercetools_filter.cache_life_time%']
\ No newline at end of file
+      arguments:
+          - '@best_it_commercetools_filter.request.client'
+          - '@cache.app'
```

Comment:
Beim Cache fände ich ein Alias über die Config schon fast besser, da hier der Use-Case zu hart eingeschränkt wird.

Reply:
Ja ... aber schon quasi "semi" zu spät. Dieser Service der den Cache nutzt wurde vor meinen Änderungen implementiert. Ich habe den Service lediglich in einen Listener "verschoben". Werde über die Config einen Cache Id mitgeben lassen (Stichwort alias). Als "default" muss aber dennoch `@cache.app` gesetzt sein, sonst haben wir einen BC (schon wieder).

Das passt wiederum zu meinen Punkt oben, dass das FilterBundle eigentlich zu viel macht ...